### PR TITLE
kodi addon checker

### PR DESCRIPTION
**Pull request by mediaminister**
Changes to meet kodi-addon-checker test at https://github.com/xbmc/repo-plugins/pulls

### CODE REVIEW

File: plugin.video.vrt.nu/addon.xml [hunk 1]
```diff
@@ -16,9 +16,7 @@
     <extension point="xbmc.python.pluginsource" library="addon.py">
         <provides>video</provides>
     </extension>
-
-    <extension point="xbmc.service" library="service.py" start="startup" />
```
Comment (dagwieers) [inline plugin.video.vrt.nu/addon.xml]: Do we remove `start="startup"` here on purpose ?

Comment (mediaminister) [reply plugin.video.vrt.nu/addon.xml]: Yes, cause this option doesn't exist anymore in Kodi Leia: look at the bottom of the page at https://kodi.wiki/view/Service_add-ons

Comment (dagwieers) [reply plugin.video.vrt.nu/addon.xml]: Ok, but it does for Krypton ? Because we still release for Krypton and Leia.

Comment (dagwieers) [reply plugin.video.vrt.nu/addon.xml]: Ok, so I guess we don't need it even when it meant something ?

Comment (mediaminister) [reply plugin.video.vrt.nu/addon.xml]: Correct, we don't need it. Everything works as expected without it.

**Pull request merged**